rust/monovertex/src/metrics.rs

```diff
@@ -29,8 +29,8 @@ use prometheus_client::registry::Registry;
 
 // Define the labels for the metrics
 // Note: Please keep consistent with the definitions in MonoVertex daemon
-pub const MONO_VERTEX_NAME_LABEL: &str = "mvtx_name";
-pub const REPLICA_LABEL: &str = "mvtx_replica";
+const VERTEX_NAME_LABEL: &str = "mvtx_name";
```

Comment:
This can stay as MONO_VERTEX_NAME_LABEL -> even though under monovertex directory, just to keep the logical difference between VERTEX and MONOVERTEX